tests/unit/IOUUtilsTest.js

```diff
@@ -24,7 +24,7 @@ function createIOUReportAction(type, amount, currency, isOffline = false, IOUTra
         iouReport.reportID,
     );
 
-    // Default is to create requests offline, if this is specified then we need to remove the pendingAction
+    // Default is to create requests online, if `isOffline` is specified then we need to remove the pendingAction
     moneyRequestAction.pendingAction = isOffline ? CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD : null;
```

Comment:
```suggestion
    // Default is to create requests online, if `isOffline` is not specified then we need to remove the pendingAction
    if (!isOffline) {
        moneyRequestAction.pendingAction = null;
    }
```

The pendingAction is already "add", let's just remove it if the requests are made online.